packages/api/src/logger/index.ts

```diff
@@ -210,103 +207,88 @@ export const defaultLoggerOptions: LoggerOptions = {
+
+export interface RedwoodLoggerOptions<TransportOptions = Record<string, any>> {
+  options?: TransportOptions
+  targets?: readonly P.TransportTargetOptions<Record<string, any> | undefined>[]
   showConfig?: boolean
 }
 
 /**
  * Creates the logger
  *
  * @param options {RedwoodLoggerOptions} - Override the default logger configuration
- * @param destination {DestinationStream} - An optional destination stream
+ * @param targets {TransportMultiOptions} - An optional Transports
  * @param showConfig {Boolean} - Show the logger configuration. This is off by default.
- *
+ *s
  * @example
  * // Create the logger to log just at the error level
  * createLogger({ options: { level: 'error' } })
  *
  * @example
  * // Create the logger to log to a file
- * createLogger({ destination: { 'var/logs/redwood-api.log' } })
+ * createLogger({ targets: [{
+      level: 'info',
+      target: 'pino-pretty', // must be installed separately
+    }]
+  })
  *
  * @return {BaseLogger} - The configured logger
  */
 export const createLogger = ({
+  targets,
   options,
-  destination,
   showConfig = false,
-}: RedwoodLoggerOptions): BaseLogger => {
-  const hasDestination = typeof destination !== 'undefined'
-  const isFile = hasDestination && typeof destination === 'string'
-  const isStream = hasDestination && !isFile
-  const stream = destination
-
-  // override, but retain default pretty print options
-  if (isPretty && options && options.prettyPrint) {
-    const prettyOptions = {
-      prettyPrint: {
-        ...(defaultLoggerOptions.prettyPrint as PrettyOptions),
-        ...(options.prettyPrint as PrettyOptions),
-      },
-    }
-
-    delete options.prettyPrint
-
-    options = {
-      ...defaultLoggerOptions,
-      ...prettyOptions,
-      ...options,
-    }
-  } else {
-    options = { ...defaultLoggerOptions, ...options }
-  }
+}: RedwoodLoggerOptions): P.BaseLogger => {
+  // TODO:
+  // Add PrettyPrintOptions
+  // Check if isFile
+  // Keep config levels and redactions
+  // Find a better default end that pretty
 
   if (showConfig) {
     console.log('Logger Configuration')
     console.log(`isProduction: ${isProduction}`)
     console.log(`isDevelopment: ${isDevelopment}`)
     console.log(`isTest: ${isTest}`)
     console.log(`isPretty: ${isPretty}`)
-    console.log(`isFile: ${isFile}`)
-    console.log(`isStream: ${isStream}`)
     console.log(`logLevel: ${logLevel}`)
     console.log(`options: ${JSON.stringify(options, null, 2)}`)
-    console.log(`destination: ${destination}`)
+    console.log(`targets: ${targets}`)
   }
-
-  if (isFile) {
-    if (isProduction) {
-      console.warn(
-        'Please make certain that file system access is available when logging to a file in a production environment.'
-      )
-    }
-
-    return pino(options, stream as DestinationStream)
+  if (isPretty) {
+    const transport = pino.transport({
+      target: 'pino-pretty',
+      options: { destination: 1 },
```

Comment:
That's one of the reasons I would like to rename destination for target. 

> The options.destination property may also be a number to represent a file descriptor. Typically this would be 1 to write to STDOUT or 2 to write to STDERR. If options.destination is not set, it defaults to 1 which means logs will be written to STDOUT.

https://github.com/pinojs/pino/blob/master/docs/transports.md#pinopretty